extensions/functions_datetime.yaml

```diff
@@ -32,7 +32,8 @@ scalar_functions:
       * SECOND Return the second (0-59).
       * MILLISECOND Return number of milliseconds since the last full second.
       * MICROSECOND Return number of microseconds since the last full millisecond.
-      * SUBSECOND Return number of microseconds since the last full second of the given timestamp.
+      * NANOSECOND Return number of nanoseconds since the last full microsecond.
+      * SUBSECOND Return number of nanoseconds since the last full second of the given timestamp.
```

Comment:
Are we changing the behavior of SUBSECOND here?  Would it be safer to add a new option and deprecate SUBSECOND?

Reply:
Yeah, deprecating might be a better.  Should we have a process to be followed for deprecating an option value?

Reply:
If the standard definition uses millisecond for SUBSECOND then we might need to leave this version permanently.